.golangci.yaml

```diff
@@ -0,0 +1,39 @@
+run:
+  timeout: 30m
+  skip-files:
+    - "^zz_generated.*"
+
+issues:
+  max-same-issues: 0
+  # Excluding configuration per-path, per-linter, per-text and per-source
+  exclude-rules:
```

Comment:
I think we need to re-do https://github.com/kubernetes/kubernetes/pull/106418/files

Reply:
>   skip-files:
    - "^zz_generated.*
?

Reply:
lines 3 and 4

Reply:
that excludes *all* linters, surely we should just be excluding staticcheck?

perhaps this needs a seperate discussion, I think it is wrong not to sanity check our generated code for defects.

Reply:
You know more than me here 😁